Remove misleading reference to {once: true} from astro:page-load example (#9121)
withastro/docs: src/content/docs/en/guides/view-transitions.mdx

The `<ViewTransitions />` component fires this event both on initial page navigation for a pre-rendered page and on any subsequent navigation, either forwards or backwards.

> You can use this event to run code on every page navigation. That is especially useful when it comes to setting up event listeners that would otherwise be lost during navigation.

Reviewer: I think we can turn the second sentence into an example:

```suggestion
You can use this event to run code on every page navigation, for example to set up event listeners that would otherwise be lost during navigation.
```

```astro
<script>
  document.addEventListener('astro:page-load', () => {
    // This runs on first page load and after every navigation.

>     setupStuff(); // e.g. add click event listeners

Reviewer: I guess it doesn’t **have** to be click listeners, so we can simplify a bit here:

```suggestion
    setupStuff(); // e.g. add event listeners
```
